classes/core-image-pelux.bbclass

```diff
@@ -40,3 +40,4 @@ toolchain_create_sdk_env_script_append() {
 IMAGE_ROOTFS_SIZE ?= "1000000"
 IMAGE_FSTYPES ?= "ext3 sdcard"
 IMAGE_FSTYPES += "ext3.gz"
```

Comment:
Do we need this filetype at all?

Reply:
I need the ext3.gz for SWUpdate artifacts

Reply:
@FlorentRevest, is it is okay to remove .tar.bz2? Or you will need this image format as well?

Reply:
@SKumarDas you can remove it, I don't need that one

Reply:
@breakreturn Could you please conform what image format you use in qemo? Do you use tar.bz2 in any case?

Reply:
@SKumarDas Maybe @jonte or @erikboto can support with this.